### PR TITLE
[12.0][MIG] web_ir_actions_act_window_message

### PR DESCRIPTION
**Pull request by Abranes**
Migration to 12.0.

cc @ioans73 @fuentes010

### CODE REVIEW

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js [hunk 1]
```diff
@@ -7,12 +7,17 @@ odoo.define('web.web_ir_actions_act_window_message', function(require)
...
         core = require('web.core'),
-        Model = require('web.Model'),
         Dialog = require('web.Dialog');
 
     var _t = core._t;
 
     ActionManager.include({
+        _handleAction: function (action, options) {
+            if (action.type === 'ir.actions.act_window.message') {
+                return this.ir_actions_act_window_message(action, options);
```
Comment (hbrunn) [inline web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: you could consider renaming this to `_execute...` to be in line with core

**Review by fuentes73 — APPROVED** [review]

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                        $content: $('<div>', content_properties),
+                        buttons: buttons.concat(
+                            this.ir_actions_act_window_message_get_buttons(
+                                action, function() { dialog.close() })
+                        ),
+                    },
+                    options)
+            )
+            dialog.open();
+            return $.when();
```
Comment (hbrunn) [inline web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: it's probably better if we return https://github.com/OCA/OCB/blob/12.0/addons/web/static/src/js/core/dialog.js#L56 here

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js (hunk 1 above)
Comment (Abranes) [reply web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: Done!

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js (hunk 2 above)
Comment (Abranes) [reply web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: Done!

Comment (hbrunn) [reply web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: thanks, but I don't see this in the diff. I'd expect something like `return dialog.open()._opened`

**Review by hbrunn — APPROVED** [review]
thanks!

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js (hunk 2 above)
Comment (Abranes) [reply web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: Sorry, I was wrong before.

**Comment by Abranes** [comment]
Is it necessary to make any more changes before merge?

Thanks!

**Comment by hbrunn** [comment]
not for me, now we just need another review

**Review by Saphery — APPROVED** [review]

**Review by willer-mo — APPROVED** [review]

**Comment by hbrunn** [comment]
...by a person with writing permissions

**Pull request merged**